### PR TITLE
Shipping method change key cost to price

### PR DESCRIPTION
**Pull request by SirDomin**
| Q               | A
| --------------- | -----
| Branch?         | 1.9
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | yes
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: UPGRADE-1.9.md [hunk 1]
```diff
@@ -200,6 +200,7 @@
     +       - { path: '^/api/v1/.*', priorities: ['json', 'xml'], fallback_format: json, prefer_extension: true }
     ```
 
+1. API CartShippingMethod key `cost` has been changed to `price`.
```
Comment (GSadee) [inline UPGRADE-1.9.md:203]:
```suggestion
1. API CartShippingMethod key `cost` has been changed to `price`.

```

**Review by GSadee — APPROVED** [review]

**Comment by AdamKasp** [comment]
is it enough? I thought that here we want to move logic from `src/Sylius/Bundle/ApiBundle/View/CartShippingMethod.php` to serializer, like in price on productVariant

**Comment by AdamKasp** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "shipping-method-price"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Review by arti0090 — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, @SirDomin! :1st_place_medal:

**Pull request merged**